App/pom.xml

```diff
@@ -241,11 +239,11 @@
             <artifactId>ChatListView</artifactId>
             <version>1.1.0-SNAPSHOT</version>
         </dependency>
-        <!--<dependency>
```

Comment:
This causes the build to fail.
`Could not find artifact org.scenic-view:scenic-view:jar:11.0.2 `
I often include ScenicView during development, but remove it when pushing. This might benefit from a long-term solution where we publish SV aritfacts.

Short-term, I think it's best not to include it when pushing